index.html

```diff
@@ -15,7 +15,34 @@
     <link href="assets/css/bootstrap.min.css" rel="stylesheet" />
     <link href="assets/css/material-kit.css" rel="stylesheet" />
   </head>
-  <body class="landing-page">
+  <body class="landing-page" style="overflow: scroll">
+    <script>
+      function onDaf() {
+        window.scrollTo(0,0)
```

Comment:
`var` should be before actions, unless there is a good reason the action should be before the `var`.